terraform/environments/ppd/main.tf

```diff
@@ -0,0 +1,22 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+terraform {
+  backend "s3" {
+    bucket = "terraformer-remote-states-use1"
+    key    = "apps/magine/ppd.tfstate"
```

Comment:
hmm. This would place it in the `/apps/` folder that is being deprecated as we move to the terra grunt layout, right?

There's a risk it might get deleted if we 'cleaned up that folder' later.

I would prefer `/magine/ppd.tfstate` personally

Reply:
I get it though that all the other clusters are already there.

Reply:
Yeah we can make a Jira ticket to move this into the terraform repo. This was the easiest place to do it with it being part of the CI pipeline for actually deploying the Lambda.

Reply:
https://maisonette.atlassian.net/browse/TEC-636